configure.ac

```diff
@@ -5,6 +5,12 @@ AC_PROG_CC
 
 AC_PATH_PROG(APXS, 'apxs')
 
+if test x"$APXS" = x"" ; then
+  AC_CHECK_FILE([/etc/redhat-release],
```

Comment:
While I appreciate the effort, 1) is this reliable as an indication of 'yum', and 2) is it worth the extra trouble rather than just saying "install <x> on rpm systems or <y> on deb systems"?